.gitignore

```diff
@@ -1,6 +1,5 @@
 .nextflow*
```

Comment:
When I ran the workflow, a folder called `null/pipeline_info` was created with execution reports. Should this be added to `.gitignore`?

Reply:
I updated the default `outdir` value to `results/`, which is included in the `.gitignore` file and is more understandable than `null`.